src/reducers/loader.js

```diff
@@ -0,0 +1,15 @@
+const START_LOADER = 'START_LOADER'
```

Comment:
related to both my comments on the `book` reducer as well as in the `action/index.js` file, let's

1. Have a separate file that houses all the action constants
2. Let's create action generators corresponding to those action constants (in the action folder).